tests/benchmarks/test_execute.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+import pytest
+
+import strawberry
+from strawberry.graphql import execute
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "items", [25, 100, 250],
+)
+async def test_execute(aio_benchmark, items):
+    @strawberry.type
+    class Patron:
+        id: int
+        name: str
+        age: int
+
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        async def patrons(self, info) -> List[Patron]:
+            return [Patron(id=i, name="Patrick", age=100) for i in range(items)]
+
+    schema = strawberry.Schema(query=Query)
+
+    query = """
+        query something{
+          patrons {
+            id
+            name
+            age
+          }
+        }
+    """
+    result = aio_benchmark(execute, schema, query)
+    assert len(result.data["patrons"]) == items
+    assert not result.errors
```

Comment:
Nice, I wonder, do you know any tool where we can see performance over time? or even compare a PR with master?

Reply:
there are tools for history yes. 
For comparison to master, I guess we could add a performance check? that compares both branches and publish it as a comment?

Reply:
https://github.com/rhysd/github-action-benchmark

Reply:
oh, that action looks nice, maybe we can integrate it another PR 😊

Reply:
im gonna add some extra fields (DateTimeField, scalars etc) and then this can be merged